data/campaigns/atl02.wmf/scripting/texts.lua

```diff
@@ -0,0 +1,458 @@
+         _([[Sidolus, I agree with you. However I think we have no choice so we need to obey to this demand. It is your obligation now to run our small people until I get back.]])
+         .. paragraphdivider() ..
+         _([[First you need to upgrade our scout’s house to a trading post. Then you need to send the demanded tributes of gold, iron, coal and logs.]]))
+         .. new_objectives(obj_trading_post)
+   }
+}
+
+trading = {
+   {
+      title = _"Trading post built",
+      body = loftomor(
+         -- TRANSLATORS: loftomor
+         _([[Sidolus, may Satul warm you! We have finished the new trading post. Hopefully this will get us back Jundlina soon]]))
+   },
+   {
+      title = _"Sidolus Agrees",
+      body = sidolus(
+         -- TRANSLATORS: sidolus
+         _([[Well done, Loftomor, well done. May Satul warm our princess and our whole people. Now we need to begin sending goods to this barbarian.]])
+         .. paragraphdivider() ..
+         _([[I think we should start with the wood deliveries as we could afford them the easiest. But we need to find more metals.]]))
+         .. new_objectives(obj_tribute)
+   }
+}
+
+tribute_not_started = {
+   {
+      title = _"Urgent Tribute Demand",
+      body = kalitath(
+         -- TRANSLATORS: Kalitath
+         _([[Strangers, we haven't received any help yet. So if you are really on our side you should start sending some wares urgently.]])..
+         _([[As we are facing great losses due to your lazyness, you should now send us double the amount of each ware as before.]])
+         .. paragraphdivider() ..
+         _([[I am sure you won't loose our alliance, wouldn't you?]]))
+   },
+   {
+      title = _"Blackmail!",
+      body = sidolus(
+         -- TRANSLATORS: Sidolus
+         _([[May Satul warm us all. Our so-called ally is blackmailing us and we can't punish his insult as he has our princess at his hands.]])
+         .. paragraphdivider() ..
+         _([[Our only chance is to double our efforts and start delivering something soon.]]))
+         .. new_objectives(obj_tribute2)
+   },
+}
+
+tribute_started = {
+   {
+      title = _"Tribute Started",
+      body = kalitath(
+         -- TRANSLATORS: Kalitath
+         _([[Strangers, we haven't received the first batch of wares. So we may finally trust you.]])..
+         _([[However our enemy in the north is very strong. I am afraid without significant help from your side we might loose the war despite any ware help you might provide.]])
+         .. paragraphdivider() ..
+         _([[They name themselves an empire and their leader is named Maletus. I want you to train your soldiers and help us defeat his lousy empire.]]))
+         .. new_objectives(obj_defeat_maletus)
+   },
+   {
+      title = _"Be Vigilant!",
+      body = jundlina(_"Jundlina",
+         -- TRANSLATORS: Jundlina
+         _([[May Satul warm you, Sidolus. Our ally now has gained some trust, but we shouldn't trust to much in them.]])
```

Comment:
I don't understand this. Who trusts whom how much, and how much trust in which direction is Jundlina advising?

Reply:
maybe due to the copy paste above. the barbarian pretends to have gained trust, but Jundlina does not trust him too much and advises to her representative to be careful and better be strong